pkg/flowkit/config/json/emulator_test.go

```diff
@@ -42,3 +43,20 @@ func Test_ConfigEmulatorSimple(t *testing.T) {
 	assert.Equal(t, emulators[0].Name, "default")
 	assert.Equal(t, emulators[0].Port, 9000)
 }
+
+//Emulator config will be empty when first initialized, when parsed from json to config, config should contain default
+//emulator
+func Test_ConfigEmulatorDefault(t *testing.T) {
+	b := []byte(`{
+	 }`)
+
+	var jsonEmulators jsonEmulators
+	err := json.Unmarshal(b, &jsonEmulators)
+	assert.NoError(t, err)
+
+	emulators, err := jsonEmulators.transformToConfig()
+	assert.NoError(t, err)
+
+	assert.Equal(t, emulators[0].Name, config.DefaultEmulatorConfigName)
+	assert.Equal(t, emulators[0].Port, config.DefaultEmulatorPort)
+}
```

Comment:
If emulator field is empty (which will be by default after this change is merged), config will have default values for emulator when parsing empty emulator field in json.